testing/cucumber/features/linters/unique_scenario_names.feature

```diff
@@ -0,0 +1,124 @@
+Feature: Unique scenario names
+
+  As a reader of documentation
+  I want each scenario to have a unique name within the same feature file, even if they were created using a Scenario Outline or Rule.
+  So that each scenario describes a specific aspect of the application's functionality.
+
+  Scenario: Linting (No Duplicates within Feature including Rules)
+    Given the following feature:
+      """
+      Feature: Sample Feature
+
+        Scenario: Unique Scenario Name 1
+          Given something
+
+        Scenario: Unique Scenario Name 2
+          Given something else
+
+        Scenario Outline: Unique Scenario Outline Name With <input>
+        Examples:
+          | input          |
+          | something      |
+          | something else |
+
+        Rule: Example Rule
```

Comment:
Lets add `Unique Scenario Name 1` to Rule. And lets add two rules with the same scenario names.

Reply:
Done